pkgs/development/python-modules/libvirt/5.9.0.nix

```diff
@@ -1,11 +1,11 @@
-{ stdenv, buildPythonPackage, fetchgit, pkgconfig, lxml, libvirt, nose }:
+{ stdenv,python3Packages, fetchgit, pkgconfig, lxml, libvirt, nose }:
 
-buildPythonPackage rec {
+python3Packages.buildPythonPackage rec {
```

Comment:
I thought the only reason to keep `5.9.0` was its compatibility with `python2`:
https://github.com/NixOS/nixpkgs/blob/b80edca6be70b13c097c58e3ad7a17034b536d49/pkgs/top-level/python-packages.nix#L6415-L6420
cc @wedens @Izorkin

Reply:
Yeah, the whole purpose of keeping 5.* around is python 2 compatibility (e.g. nixops still uses python 2).